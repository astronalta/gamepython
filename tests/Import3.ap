import Sub2


Import3 < Object {
    var1 = Sub1::Test()
    var2 = Sub1::test()
    var3 = Sub2::Test()
    var4 = Sub2::test()
    var5 = Test()
    var6 = test()
}
