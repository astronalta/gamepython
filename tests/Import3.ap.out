Module
  name: Sub1
  feature0: Class
    name: Test
    comment:
  feature1: Function
    name: test
    type: Int
    comment:
    block: Block
      child0: Return
        expression: IntegerLiteral(0)
  feature2: Function
    name: bob
    type: Void
    comment:
    block: Block
Module
  name: Sub2
  feature0: Class
    name: Test
    comment:
  feature1: Function
    name: test
    type: Int
    comment:
    block: Block
      child0: Return
        expression: IntegerLiteral(0)
Module
  name: 
  feature0: Class
    name: Import3
    comment:
    feature0: Attribute
      name: var1
      type: <<notype>>
      initializer: Construct
        type: Sub1::Test
    feature1: Attribute
      name: var2
      type: <<notype>>
      initializer: Call
        name: test
        module: Sub1
    feature2: Attribute
      name: var3
      type: <<notype>>
      initializer: Construct
        type: Sub2::Test
    feature3: Attribute
      name: var4
      type: <<notype>>
      initializer: Call
        name: test
        module: Sub2
    feature4: Attribute
      name: var5
      type: <<notype>>
      initializer: Construct
        type: Test
    feature5: Attribute
      name: var6
      type: <<notype>>
      initializer: Call
        name: test
